tests/org.eclipse.ui.tests.dnd/src/org/eclipse/ui/tests/dnd/DragTestSuite.java

```diff
@@ -0,0 +1,218 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Ali Muhsin KÖKSAL.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * Contributors:
+ *     Ali Muhsin KÖKSAL - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.ui.tests.dnd;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Shell;
+import org.eclipse.ui.IPageLayout;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.AllTests;
+
+import junit.framework.Test;
+
+@RunWith(AllTests.class)
+@Ignore("DND support not implemented yet in E4")
```

Comment:
You need to remove the `@Ignore` and should set the test suite as the class executed for the bundle in the `build.properties` like in the other test bundles, e.g.:
https://github.com/eclipse-platform/eclipse.platform.ui/blob/223dfe7383547fdead8d59f43fa7d8ad6b2e2022/tests/org.eclipse.ui.tests/build.properties#L32

We finally also need to add a proper `test.xml` to have the tests executed in Jenkins.